packages/components/ng-ovh-line-diagnostics/src/steps/solution-proposal-step.html

```diff
@@ -5,12 +5,12 @@
         src="{{:: $ctrl.assets.HUMAN_GOOD }}"
         width="180"
         height="180"
-        alt=""
+        alt
```

Comment:
Removing alt empty var? It should been kept no?

Reply:
Yes, I've spotted them. But it's done by the updater. It does the same with our `href=""` (or any empty attribute).